hydromt_wflow/wflow.py

```diff
@@ -768,7 +768,7 @@ def setup_lakes(self, lakes_fn="hydro_lakes", min_area=10.0):
             "input.lateral.river.lake.waterlevel": "LakeAvgLevel",
         }
 
-        if lakes_fn not in ["hydro_lakes"]:
+        if lakes_fn not in self.data_catalog:
```

Comment:
Should we throw a warning here? Later the `get_geodataframe` method will return an error if the dataset is not found which is more consistent with other methods.

Reply:
idem for glaciers in Line 1124